packages/@okta/vuepress-site/docs/guides/custom-url-domain/enable-the-custom-domain/index.md

```diff
@@ -10,18 +10,25 @@ Use the configuration wizard to walk through the steps to customize your Okta UR
     * The **Update Certification** button appears if a custom URL domain is already configured for your org. To delete the current configuration, click **Restore to default**.
 
 ### Add your subdomain information
-On the Domain page of the configuration wizard, enter your subdomain name, for example, `login.example.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
+On the Domain page of the configuration wizard, enter your subdomain name, for example, `login.exampledomain.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
```

Comment:
The `exampledomain.com` domain is not owned by anybody and we _must not_ use it here unless Okta purchases that domain. Otherwise, we must restrict ourselves to the domains or TLDs listed in [RFC 2606](https://tools.ietf.org/html/rfc2606#page-2)

Reply:
I agree with @jpf. I think `example.com` should be used.

Reply:
This is my bad. I had `example.com` when I originally created the doc. I had some momentary lapse of reason yesterday afternoon and thought that adding domain was a brilliant idea. NOT. I'll fix these. Thank you.